contributing.md

```diff
@@ -1,3 +1,12 @@
+#### How to reach us
+The mailing list for developers : systemroles@lists.fedorahosted.org
+
+[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+
+If you are using IRC, join the `#systemroles` IRC channel on freenode (it is rather new,
```

Comment:
Thank you. Please remove the remark about it being rather new, there are now several contributors.